### ISSUE TITLE
Feature Request: Scroll Volume Control

### ISSUE THREAD

**Issue by bigmutty** [OptiKey/OptiKey#704]
I think it would be neat if there was a dynamickey action that brought up the look to scroll feature in the center of the screen but instead of scrolling the window content, it adjusted the volume. An On-screen-display of the volume level could also be displayed. Of course the speed and acceleration could be customizable.

**Comment by JuliusSweetland** [comment]
Would it not be more straightforward to have volume up / down keys on a dynamic keyboard?

**Comment by bigmutty** [comment]
Yes, that would be more straightforward, but what would be a good increment amount? Sometimes I want to adjust the volume just a small amount and other times I want to crank it up. I know, I'm a bit of a control freak.

**Comment by kmcnaught** [comment]
FYI in a separate fork (not yet public) I've added some functionality which effectively copies the "look to scroll" feature but in a simpler form, and lets you hook up new callbacks to do whatever you want with the X,Y positions. I'm using this to control emulated Xbox joysticks, but it's very generic and could be used to add this functionality. I'm not sure quite what you'd do to give on-screen display for feedback, but the actual volume-changing would slot in trivially. 

This isn't ready to be contributed yet, but I'll let you know when it's tidied up a bit in case it's helpful.

**Comment by bigmutty** [comment]
Thanks, @kmcnaught. That sounds cool! For the time being I've found that I can get near the desired outcome by placing the "look-to-scroll" over a volume gadget I have in my sidebar.
